test/e2e/maintenance-event-cancellation-test

```diff
@@ -80,7 +94,9 @@ helm upgrade --install $CLUSTER_NAME-emtp $SCRIPTPATH/../../config/helm/ec2-meta
   --set ec2MetadataTestProxy.enableScheduledMaintenanceEvents="true" \
   --set ec2MetadataTestProxy.enableSpotITN="false" \
   --set ec2MetadataTestProxy.scheduledEventStatus="canceled" \
-  --set ec2MetadataTestProxy.port="$IMDS_PORT"    
+  --set ec2MetadataTestProxy.port="$IMDS_PORT" \
```

Comment:
Since the worker node is now tainted, the proxy won't schedule there unless it tolerates the taint.